src/sentry/api/urls.py

```diff
@@ -2063,6 +2064,11 @@
         SentryAppComponentsEndpoint.as_view(),
         name="sentry-api-0-sentry-app-components",
     ),
+    url(
+        r"^sentry-apps/(?P<sentry_app_slug>[^\/]+)/avatar/$",
+        SentryAppAvatarEndpoint.as_view(),
+        name="sentry-api-0-sentry-app-avatar",
+    ),
```

Comment:
@ceorourke Is there an endpoint for uploading logos when creating sentry apps? New sentry apps won't have a `sentry_app_slug` immediately

Reply:
For posterity, we're solving this by having the upload fields appear after the initial save - it's a little wonky but we do the same for API tokens so it's not unprecedented.